test/integration/client-side-operations-timeout/client_side_operations_timeout.spec.test.ts

```diff
@@ -18,6 +20,11 @@ const cursorOperations = [
   'listCollectionNames'
 ];
 
+const bulkWriteOperations = [
+  'timeoutMS applies to whole operation, not individual attempts - bulkWrite on collection',
+  'timeoutMS applies to whole operation, not individual attempts - insertMany on collection'
```

Comment:
Skipped for now. We seem to be throwing the TimeoutError at the correct place, but it's getting wrapped by the BulkWriteError